eng/common/scripts/Detect-Api-Changes.ps1

```diff
@@ -0,0 +1,98 @@
+[CmdletBinding()]
+Param (
+  [Parameter(Mandatory=$True)]
+  [string] $ArtifactPath,
+  [Parameter(Mandatory=$True)]
+  [string] $PullRequestNumber,
+  [Parameter(Mandatory=$True)]
+  [string] $BuildId,
+  [Parameter(Mandatory=$True)]
+  [string] $CommitSha,
+  [Parameter(Mandatory=$True)]
+  [string] $ArtifactList,
+  [string] $ArtifactName = "packages",
+  [string] $APIViewUri = "https://apiview.dev/PullRequest/DetectApiChanges"
+)
+
+# Submit API review request and return status whether current revision is approved or pending or failed to create review
+function Submit-Request($filePath)
+{
+    $repoName = "azure-sdk-for-$Language"
+    $queryParam = "artifactName=$ArtifactName&buildId=$BuildId&filePath=$filePath&commitSha=$CommitSha&repoName=$repoName&pullRequestNumber=$PullRequestNumber"    
+    $uri= "$($APIViewUri)?$($queryParam)"
+    Write-Host "Request URI: $uri"
+    try
+    {
+        $Response = Invoke-WebRequest -Method 'GET' -Uri $uri
```

Comment:
I think something like this is a little more readable and less error prone:

```suggestion
    $query = [System.Web.HttpUtility]::ParseQueryString('')
    $query.Add('artifactName', $ArtifactName)
    $query.Add('buildId', $BuildId)
    $query.Add('filePath', $filePath)
    $query.Add('commitSha', $CommitSha)
    $query.Add('repoName', $repoName)
    $query.Add('pullRequestNumber', $PullRequestNumber)
    $uri = [System.UriBuilder]$APIViewUri
    Write-Host "Request URI: $($uri.Uri.OriginalString)"
    try
    {
        $Response = Invoke-WebRequest -Method 'GET' -Uri $uri.Uri
```

Reply:
done